lib/infra/quests/db.ex

```diff
@@ -94,28 +87,4 @@ defmodule Infra.Quests.Db do
         nil
     end
   end
-
-  defimpl Projectionist.Reader do
```

Comment:
More relics of the past as we were first setting up projectionist and trying to figure out where to put the reader logic. See `QuestReader` in this PR.